functional/FunctionalTest.java

```diff
@@ -2244,24 +2245,17 @@ public static void presignedGetObject_test2() throws Exception {
         throw new Exception("object content differs");
       }
       client.removeObject(RemoveObjectArgs.builder().bucket(bucketName).object(objectName).build());
-      mintSuccessLog(
-          "presignedGetObject(PresignedGetObjectArgs args)", "expiry :3600 sec ", startTime);
+      mintSuccessLog(methodName, "presigned get object expiry :3600 sec ", startTime);
     } catch (Exception e) {
-      mintFailedLog(
-          "presignedGetObject(PresignedGetObjectArgs args)",
-          "expiry : 3600 sec",
-          startTime,
-          null,
-          e.toString() + " >>> " + Arrays.toString(e.getStackTrace()));
-      throw e;
+      handleException(methodName, "presigned get object expiry :3600 sec ", startTime, e);
     }
   }
 
-  /** public String presignedGetObject(PresignedGetObjectArgs args). */
-  public static void presignedGetObject_test3() throws Exception {
+  /** public String getPresignedObjectUrl(GetPresignedObjectUrlArgs args). */
+  public static void getPresignedObjectUrl_test3() throws Exception {
+    String methodName = "getPresignedObjectUrl(GetPresignedObjectUrlArgs args)";
```

Comment:
Please add an `args` variable similar to the previous suggestions and use it in the logging / exception-handling method calls.